package.json

```diff
@@ -99,6 +99,8 @@
     "@leafygreen-ui/tooltip": "9.1.6",
     "@leafygreen-ui/typography": "16.4.1",
     "@rjsf/core": "4.2.0",
+    "@sentry/react": "7.56.0",
```

Comment:
Is there something we need to do to support replays? I didn't see them populated in sentry and its one of Sentry's coolest features

Reply:
Let's limit this ticket to POC work. Created EVG-20308